S3.Rmd

```diff
@@ -461,6 +463,7 @@ foo.character <- function(x, ..., truncate = 10) {
 foo("This is a long string", trncate = 2)
 ```
 
+<!-- FIXME: but checking that ... is empty defeats the 'rationale' for ... in the first place -- to allow arguments to be passed to next methods, in 'your' method or some other method on the same generic -->
```

Comment:
Oh shoot, you're right. I forgot about that problem.

I'm still fairly convinced that you `...` should be the default, otherwise you fundamentally limit the extensibility of the methods. But silently eating misnamed arguments is a big problem. I don't think `NextMethod(...)` will work because it modifies the existing call.

But I definitely need a more nuanced discussion here